eng/pipelines/common/xplat-setup.yml

```diff
@@ -12,6 +12,13 @@ jobs:
 - template: ${{ coalesce(parameters.helixQueuesTemplate, parameters.jobTemplate) }}
   parameters:
     variables:
+      - name: _containsCoreClrChange
+        value: $[ dependencies.checkout.outputs['SetPathVars_coreclr.containschange'] ]
+      - name: _containsLibrariesChange
+        value: $[ dependencies.checkout.outputs['SetPathVars_libraries.containschange'] ]
+      - name: _containsInstallerChange
+        value: $[ dependencies.checkout.outputs['SetPathVars_Installer.containschange'] ]
```

Comment:
Why is `Installer` capitalized while the rest are not? Feel like consistency in casing is generally important in paths  / names for build infrastructure.

Reply:
I believe it was just a typo. Will update to make it lowercase.

Reply:
Fixed.